src/NHibernate.Test/NHSpecificTest/NH750/ManyToManyThrowsForNotFoundFixture.cs

```diff
@@ -44,6 +45,7 @@ protected override void OnTearDown()
 		[Test]
 		public void LazyLoad()
 		{
+			using var log = new SqlLogSpy();
```

Comment:
It is not used by the test. Forgotten debug temporary change?

Reply:
Removed